src/Tooltip/README.md

```diff
@@ -36,3 +29,23 @@ notes: |
   ))}
 </>
 ```
+
+##### Light version
+
+```jsx live
+<div className="bg-dark-700 p-4">
+  {['top', 'right', 'bottom', 'left'].map((placement) => (
+    <OverlayTrigger
+      key={placement}
+      placement={placement}
+      overlay={
+        <Tooltip variant="tooltip-light" id={`tooltip-${placement}`}>
```

Comment:
I missed this earlier, sorry:

`<Tooltip variant="light" />` seems like a better api than `<Tooltip variant="tooltip-light" />` could we change to reflect that?

Reply:
@abutterworth fixed!